util/progress/progressui/printer.go

```diff
@@ -39,6 +39,33 @@ func (p *textMux) printVtx(t *trace, dgst digest.Digest) {
 	if !ok {
 		return
 	}
+	defer func() {
+		v.nonStatusUpdates = false
+	}()
+
+	shouldPrintStatus := make(map[string]bool)
```

Comment:
Yeah will do, this just slightly adapts pre-existing logic. Previously, the code would decide to not display a status update it there wasn't enough change from the last one, but it would only do that after already printing the vertex "header", so you'd sometimes end up seeing something like `#123 my.image:latest` but then no actual update. The change here just determines whether there is anything to print at all before printing that header.